app/common/environment.go

```diff
@@ -0,0 +1,45 @@
+package common
+
+import "os"
+
+const envVarName = "ALGOREA_ENV"
+const productionEnv = "prod"
+const developementEnv = "dev"
+const testEnv = "test"
```

Comment:
Why do you think that only these three environments are possible? I would like to be able to create separate environments for task, for example. I can have 'dev_with_many_users' + 'dev_many_teams' (each uses its own DB) which are both kind of 'dev'. So, I think we should not constrain the list of possible environments. The only "special" environment that the code should know is "test". Also, I think we don't need the IsEnvDev() function.

Reply:
IsEnvDev is for your `if environment == devEnvironment {`

Reply:
But will the IsEnvDev() return true for 'dev_with_many_users'? I think we should add a config variable 'stub_auth' for this and use it instead of `if environment == devEnvironment`

Reply:
This is what you had done. Anyway, it will change in a few days with the new auth...